zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.zeppelin.notebook;
 
+import static org.apache.commons.lang.StringUtils.*;
+
```

Comment:
static import statement is located at the end of all imports statements. Could you please move it?